kinematic_chain_base_link: torso
max_tf_attempts: 5
max_joint_pos_error: 0.1
loop_rate: 1000
left_obj_frame: left_object
right_obj_frame: right_object

workspace_limits_marker_alpha: 0.5
absolute_pose_limits:
  upper_limits: [0.6, 0.6, 0.6, 1.57, 1.57, 1.57]
  upper_thresholds: [0.50, 0.5, 0.5, 90, 90, 90]
  lower_limits: [0.3, -0.6, 0.0, -100, -100, -100]
  lower_thresholds: [0.4, -0.5, 0.1, -90, -90, -90]

relative_gain:
  data: [1.0, 0.0, 0.0, 0.0, 0.0, 0.0,
         0.0, 1.0, 0.0, 0.0, 0.0, 0.0,
         0.0, 0.0, 1.0, 0.0, 0.0, 0.0,
         0.0, 0.0, 0.0, 1.0, 0.0, 0.0,
         0.0, 0.0, 0.0, 0.0, 1.0, 0.0,
         0.0, 0.0, 0.0, 0.0, 0.0, 1.0]

eef1:
  kdl_eef_frame: left_gripper
  gripping_frame: left_gripper
  has_ft_sensor: false
  sensor_frame: ""
  sensor_topic: ""

eef2:
  kdl_eef_frame: right_gripper
  gripping_frame: right_gripper
  has_ft_sensor: false
  sensor_frame: ""
  sensor_topic: ""

secundary_gain:
  data: [10.0, 0.0, 0.0, 0.0, 0.0, 0.0,
         0.0, 10.0, 0.0, 0.0, 0.0, 0.0,
         0.0, 0.0, 10.0, 0.0, 0.0, 0.0,
         0.0, 0.0, 0.0, 10.0, 0.0, 0.0,
         0.0, 0.0, 0.0, 0.0, 10.0, 0.0,
         0.0, 0.0, 0.0, 0.0, 0.0, 10.0]

marker_servers: ['relative_motion_markers', 'absolute_motion_markers', 'secundary_motion_markers']

relative_motion_markers:
  marker_names: ['obj1', 'obj2']

  obj1:
    parent_frame: 'left_gripper'
    name: 'left_object'
    initial_pose: [0, 0, 0, -0.5, 0.5, -0.5, 0.5]

  obj2:
    parent_frame: 'right_gripper'
    name: 'right_object'
    initial_pose: [0, 0, 0, 0.5, 0.5, 0.5, 0.5]

absolute_motion_markers:
  marker_names: ['absolute_pose']

  absolute_pose:
    parent_frame: 'base'
    name: 'absolute_pose'
    initial_pose: [0.7, 0.0, 0.5, 0, 0, 0, 1]

secundary_motion_markers:
  marker_names: ['secundary_pose']

  secundary_pose:
    parent_frame: 'left_gripper'
    name: 'secundary_pose'
    initial_pose: [0.0, 0.0, 0.0, 0, 0, 0, 1]
